src/test/scala/com/johnsnowlabs/util/TrainingModelsScript.scala

```diff
@@ -0,0 +1,11 @@
+import com.johnsnowlabs.nlp.SparkAccessor
+import com.johnsnowlabs.pretrained.EnModelsTraining
+
+// ToDo 1. Downlaad datasets before if needs
+// ToDo 2. Create Uploader that merges metadata
+
+object TrainingModelsScript extends App {
+
+  val training = new EnModelsTraining(SparkAccessor.spark)
+  training.trainAllAndSave("models", true)
```

Comment:
is this writing to disk only? shouldn't we write also to S3?

Reply:
Yes, it writes to disk. I'm planning to create s3 uploader that merges metadata files and uploads resource files.